[hi] Localized multitenancy.md

oops-shlok opened the pull request:
### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

oops-shlok committed:
[hi] Localized multitenancy.md

Signed-off-by: Shlok Upadhyay <deep.upu9@gmail.com>

aj11anuj reviewed (changes requested):
Same mistake, [refer this comment](https://github.com/cncf/glossary/pull/3443#pullrequestreview-2760283037)

oops-shlok committed:
[hi] Fixed Core hindi words

Signed-off-by: Shlok Upadhyay <deep.upu9@gmail.com>

oops-shlok commented:
> Same mistake, [refer this comment](https://github.com/cncf/glossary/pull/3443#pullrequestreview-2760283037)

I have fixed it now and updated the code. Please review once.

aj11anuj commented on content/hi/multitenancy.md:
```diff
@@ -0,0 +1,33 @@
+## समस्या
+
+अगर मल्टीटेनेंसी न हो, तो हर यूज़र या कंपनी के लिए अलग-अलग सॉफ़्टवेयर इंस्टॉल करना पड़ेगा।  
+इससे ज़्यादा सिस्टम चाहिए, ज़्यादा मेंटेनेंस करना पड़ता है और खर्च भी बढ़ जाता है।
```
```suggestion
इसके लिए ज़्यादा सिस्टम चाहिए, ज़्यादा मेंटेनेंस करना पड़ेगा और खर्च भी बढ़ जाएगा।
```

aj11anuj reviewed (commented):
Change wordings like "seva", "vegyara", "mahaol" to "service", "etc", "environment". Also make sure to change links to redirect to hindi localized pages and not english ones.

mailnike reviewed (commented):
Thanks for this localization, it reads clearly and the examples are nicely localized. One suggestion for consistency with the cross-links you already added (एपीआई, क्लाइंट-सर्वर आर्किटेक्चर): in the **संबंधित बातें** section, the `SaaS` mentions could link to the software-as-a-service term, e.g. `[SaaS](/software-as-a-service/)`, mirroring how the English source cross-references related terms. Content-wise it looks accurate.
